src/layout/Header.tsx

```diff
@@ -1,54 +1,45 @@
 import FilterIcon from "@/components/svgs/FilterIcon";
-import useIsInitialStateWithoutFilter from "@/hooks/useIsInitialStateWithoutFilter";
-import { useTailwindBreakpoint } from "@/hooks/useTailwindBreakpoint";
 import useUIContext from "@/hooks/useUIContext";
 import { removeMarkdownCharacters } from "@/utils/elastic-search-ui-functions";
-import { SearchBox } from "@elastic/react-search-ui";
 import React from "react";
-import SearchBoxView from "../components/customSearchboxView/SearchBoxView";
+import SearchBoxView from "../components/customSearchbox/SearchBoxView";
 import useSearchQuery from "../hooks/useSearchQuery";
+import SearchBox from "@/components/customSearchbox/SearchBox";
 
 const Header = ({ openForm }) => {
   const { sidebarToggleManager } = useUIContext();
   const { makeQuery, filterFields, pagingInfo } = useSearchQuery();
-  const { hiddenHomeFacet } = useIsInitialStateWithoutFilter();
 
   const numberOfAppliedFilters = filterFields.length;
 
   const handleSubmit = (input: string) => {
     makeQuery(input);
   };
 
-  const isMediumScreen = useTailwindBreakpoint("md");
-
   const handleAutoCompleteSelect = (
-    selection,
-    autoCompleteData,
-    defaultFunction
+    selection
   ) => {
     if (!selection.suggestion) return;
     makeQuery(removeMarkdownCharacters(selection.suggestion));
   };
 
   return (
     <div className="flex pt-10 md:pt-0 gap-2 mx-auto max-w-3xl w-full justify-center bg-custom-background">
-      {(!isMediumScreen || !hiddenHomeFacet) && (
-        <SearchBox
-          autocompleteMinimumCharacters={3}
-          view={SearchBoxView}
-          autocompleteSuggestions={true}
-          debounceLength={0}
-          onSubmit={handleSubmit}
-          className="w-full"
-          onSelectAutocomplete={handleAutoCompleteSelect}
-        />
-      )}
+      <SearchBox
+        autocompleteMinimumCharacters={3}
+        view={SearchBoxView}
+        autocompleteSuggestions={true}
+        debounceLength={0}
+        onSubmit={handleSubmit}
+        className="w-full"
+        onSelectAutocomplete={handleAutoCompleteSelect}
```

Comment:
How is the `SearchBox` hidden for medium screen?

Reply:
It is being managed by layout
https://github.com/bitcoinsearch/bitcoinsearch-app/blob/6e84e46f3ce6df352fe5412460c2adfee7babb3c/src/layout/Layout.tsx#L21-L26

sidenote: I'm using this to remember the format to link files
`https://github.com/<username>/<repository>/blob/<branch>/<path-to-file>#L<start-line>-L<end-line>`